dev/index.d.ts

```diff
@@ -37,3 +40,60 @@ export class TestOutputChannel implements OutputChannel {
     public hide(): void;
     public dispose(): void;
 }
+
+export type Verbosity = 'debug' | 'silent' | 'normal';
+
+export interface DefaultWebpackOptions {
+    projectRoot: string;
+
+    /**
+     * Additional entrypoints besides the main 'extension' entrypoint
+     */
+    entries?: { [key: string]: string };
+
+    /**
+     * Modules that we can't easily webpack for some reason. These node modules and all their dependencies will be excluded from bundling.
+     */
+    externalNodeModules?: string[];
+
+    /** Additional external entries (externalNodeModules are added automatically) */
+    externals?: webpack.ExternalsObjectElement,
+
+    /**
+     * Additional loader module rules
+     */
+    loaderRules?: webpack.RuleSetRule[],
+
+    /**
+     * Additional plug-ins
+     */
+    plugins?: webpack.Plugin[];
+
+    /**
+     * Suppress deleting the dist folder before webpack
+     */
+    suppressCleanDistFolder?: boolean;
+
+    /**
+     * Logging verbosity
+     */
+    verbosity?: Verbosity;
+}
+
+export declare function getDefaultWebpackConfig(options: DefaultWebpackOptions): webpack.Configuration;
+
+/**
+ * Installs the azure account extension before running tests (otherwise our extension would fail to activate)
+ * NOTE: The version isn't super important since we don't actually use the account extension in tests
+ */
+export declare function gulp_installAzureAccount(): Promise<void> | Stream;
+
+/**
+ * Spawns a child process
```

Comment:
I would write a better description than this. Honestly I think a normal `cp.spawn` will work in 99% of cases and webpack is just weird (as far as I know webpack is the only case that actually needs this)

Reply:
I think the original was mainly to deal with listening for process errors. Gulp no longer needs that, and I'm not sure what needed the .cmd/exe fix, doesn't seem to repro now, so I've removed.

Reply:
Actually, still an issue (due to Windows shell behavior vs abc/abc.cmd/abc.exe/etc), but moved to only webpack code.